pages/tutorials/3.0/getting-started/compile-with-cmake.md

```diff
@@ -7,21 +7,28 @@
 
 ## Introduction
 
-Admittedly, the title of this tutorial is a bit misleading. You will not *compile* SFML with CMake, because CMake is *not a compiler*. So... what is CMake?
+Admittedly, the title of this tutorial is a bit misleading.
+You will not *compile* SFML with CMake, because CMake is *not a compiler*.
+So, what is CMake?
 
-CMake is an open-source meta build system. Instead of building SFML, it builds what builds SFML: Visual Studio solutions, Code::Blocks projects, Linux makefiles, XCode projects, etc. In fact it can generate the makefiles or projects for any operating system and compiler of your choice. It is similar to autoconf/automake or premake for those who are already familiar with these tools.
+CMake is an open-source meta build system.
+Instead of building SFML, it builds what builds SFML: Visual Studio solutions, Code::Blocks projects, Linux Makefiles, XCode projects, etc.
+In fact it can generate the Makefiles or projects for any operating system and compiler of your choice.
+It is similar to autoconf/automake or premake for those who are already familiar with these tools.
 
-CMake is used by many projects including well-known ones such as Blender, CLion, KDE, Ogre, and many more. You can read more about CMake on its [official website](http://www.cmake.org/ "CMake official website") or in its [Wikipedia article](http://en.wikipedia.org/wiki/CMake "Wikipedia page of CMake").
+CMake is used by many projects including well-known ones such as Minecraft: Bedrock Edition, LLVM, Blender, CLion, KDE, Ogre, and many more.
+You can read more about CMake on its [official website](http://www.cmake.org/ "CMake official website") or in its [Wikipedia article](http://en.wikipedia.org/wiki/CMake "Wikipedia page of CMake").
```

Comment:
```suggestion
You can read more about CMake on its [official website](https://www.cmake.org/ "CMake official website") or in its [Wikipedia article](https://en.wikipedia.org/wiki/CMake "Wikipedia page of CMake").
```

You always know the content is old, when still uses HTTP links 😄

Reply:
I fixed all non HTTPS links to external websites.